labman/db/process.py

```diff
@@ -2948,12 +2961,17 @@ def _generate_amplicon_sample_sheet(self):
         str
             The illumina-formatted sample sheet
         """
-        fixed_run_name = SequencingProcess._bcl_scrub_name(self.run_name)
-        data = (
-            'Sample_ID,Sample_Name,Sample_Plate,Sample_Well,I7_Index_ID,'
-            'index,Sample_Project,Description,,\n'
-            '%s,,,,,NNNNNNNNNNNN,,,,,' % fixed_run_name)
-        return self._format_sample_sheet(data)
+        # the "Description" => "Well_Description" change was for the
+        # compatibility with EBI submission
+        data = ['%sSample_ID,Sample_Name,Sample_Plate,Sample_Well,'
+                'I7_Index_ID,index,Sample_Project,Well_Description,,'
+                % ('Lane,' if self.include_lane else '')]
+        for pool, lane in self.pools:
+            data.append('%s%s,,,,,NNNNNNNNNNNN,,%s,,,'
+                        % (('%s,' % lane if self.include_lane else ''),
```

Comment:
Would it be possible to enclose `lane if self.include_lane else ''` in parentheses? The execution here is entirely dependent on the order of operations, and it's not clear on a static review if the evaluation is first `'%s,' % lane` which would make that "expr1" or if the evaluation is first `lane if self.include_lane else ''`.

Reply:
Good suggestion!